### PR TITLE
Mirror openshift-azure images hourly

### PR DESCRIPTION
**Pull request by 0xmichalis**
@openshift/sig-azure this cronjob sets up image mirroring between our CI registry and quay, ptal

### CODE REVIEW

**Comment by kargakis** [comment]
First run worked like a charm
```
quay.io/
  openshift-on-azure/sync
    blobs:
      registry.svc.ci.openshift.org/azure/azure-plugins sha256:286faa0d5ecff4e3eee3383fabd370f21cadf413112350655cee3750accdee5a 2.643KiB
      registry.svc.ci.openshift.org/azure/azure-plugins sha256:46326a8995ec7b3e60d366315afbb8749c75f7d78c3098b783e6f91e2673448c 8.579MiB
      registry.svc.ci.openshift.org/azure/azure-plugins sha256:cdcd8938ebf75b92148acfdea5c506ec98bde37460fec0394d7a1042b020548b 42.28MiB
      registry.svc.ci.openshift.org/azure/azure-plugins sha256:7dc0dca2b1516961d6b3200564049db0a6e0410b370bb2189e2efae0d368616f 71.23MiB
    manifests:
      sha256:fcff082373321a44bd49b3ab9ad5b7450682eebaaa89512cb1419ce9c45a37b1 -> latest
      sha256:fcff082373321a44bd49b3ab9ad5b7450682eebaaa89512cb1419ce9c45a37b1 -> v3.10
      sha256:fcff082373321a44bd49b3ab9ad5b7450682eebaaa89512cb1419ce9c45a37b1 -> v3.10.0
  stats: shared=0 unique=4 size=122.1MiB ratio=1.00

phase 0:
  quay.io openshift-on-azure/sync blobs=4 mounts=0 manifests=3 shared=0

info: Planning completed in 870ms
uploading: quay.io/openshift-on-azure/sync sha256:46326a8995ec7b3e60d366315afbb8749c75f7d78c3098b783e6f91e2673448c 8.579MiB
uploading: quay.io/openshift-on-azure/sync sha256:7dc0dca2b1516961d6b3200564049db0a6e0410b370bb2189e2efae0d368616f 71.23MiB
uploading: quay.io/openshift-on-azure/sync sha256:cdcd8938ebf75b92148acfdea5c506ec98bde37460fec0394d7a1042b020548b 42.28MiB
sha256:9778d4a76ac841e3062badeec72eb806588765fe1053543e1479d6f9f79a3fa9 quay.io/openshift-on-azure/sync:latest
sha256:a6c65ac7977b44f42fea740b857336eb0da69d71e61f62fe4f08c40017293f87 quay.io/openshift-on-azure/sync:v3.10
info: Mirroring completed in 23.75s (5.39MB/s)
sha256:7f34b08af3dfb7555362162d194719ee355059e2034c0a11c3d9d8e2670e909f quay.io/openshift-on-azure/sync:v3.10.0
```

File: projects/azure/image-mirror/image-mirror.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          metadata:
+            name: "image-mirror-openshift-azure-v3.10-quay"
+            labels:
+              app: "image-mirror-openshift-azure-v3.10-quay"
+          spec:
+            restartPolicy: OnFailure
+            volumes:
+            - name: push
+              secret:
+                secretName: openshift-on-azure-openshiftci-secret  # generate from quay console
```
Comment (mjudeikis) [inline projects/azure/image-mirror/image-mirror.yaml]: Where is this coming from?

Comment (kargakis) [reply projects/azure/image-mirror/image-mirror.yaml]: See the comment, needs to be generated from quay

Comment (mjudeikis) [reply projects/azure/image-mirror/image-mirror.yaml]: So same as with other secrets. No makefile for creation for it?

**Comment by kargakis** [comment]
Ideally we would push after each commit but cannot run this as a `run_after_success` yet because of https://github.com/kubernetes/test-infra/issues/8862 (or we should move our images in the `ci` namespace).

File: projects/azure/image-mirror/image-mirror.yaml (hunk 1 above)
Comment (jim-minter) [reply projects/azure/image-mirror/image-mirror.yaml]: glad it's temporary - will be happy to see this being push-based.

**Pull request merged**